### PR TITLE
[FLINK-34180][docs] Migrate doc website from ververica to flink

### PR DESCRIPTION
**Pull request by GOODBOY008**
Add flink cdc document link to flink website.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@leonardBang @PatrickRen  PTAL

File: docs/content.zh/getting-started/training-course.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-weight: 6
```
Comment (PatrickRen) [inline docs/content.zh/getting-started/training-course.md:2]: Is this modified by mistake?

File: docs/content.zh/getting-started/with-flink-cdc.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+weight: 6
+title: With Flink CDC
+bookHref: "https://nightlies.apache.org/flink/flink-cdc-docs-stable/docs/try-flink-cdc"
```
Comment (PatrickRen) [inline docs/content.zh/getting-started/with-flink-cdc.md:4]: This link points to an empty page

File: docs/content/getting-started/training-course.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-weight: 6
```
Comment (PatrickRen) [inline docs/content/getting-started/training-course.md:2]: This is also not necessary I think

**Review by PatrickRen — COMMENTED** [review]
@GOODBOY008 Thanks for the PR! I left some comments.

File: docs/content.zh/getting-started/training-course.md (hunk 1 above)
Comment (GOODBOY008) [reply docs/content.zh/getting-started/training-course.md:2]: @PatrickRen  **Weight** will effect sort. I insert before this catalog ,so modify it's weight.

File: docs/content.zh/documentation/flink-cdc-master.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+weight: 12
+title: Flink CDC Master (snapshot)
```
Comment (PatrickRen) [inline docs/content.zh/documentation/flink-cdc-master.md]: What about `CDC Master (snapshot)` to align with other titles?

File: docs/content.zh/documentation/flink-cdc-master.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+weight: 12
```
Comment (PatrickRen) [inline docs/content.zh/documentation/flink-cdc-master.md:2]: If we can control the order, what about resorting all titles in alphabetic order?

**Review by PatrickRen — COMMENTED** [review]
@GOODBOY008 Thanks for the update! I left some comments.

File: docs/content.zh/getting-started/training-course.md (hunk 1 above)
Comment (PatrickRen) [reply docs/content.zh/getting-started/training-course.md:2]: Ah thanks for the explanation. I was thinking it was related to font lol.

File: docs/content.zh/getting-started/with-flink-cdc.md (hunk 2 above)
Comment (PatrickRen) [reply docs/content.zh/getting-started/with-flink-cdc.md:4]: Had an offline discussion with @GOODBOY008 and we decided to add a page on CDC side, so we keep the link as is.

**Review by PatrickRen — APPROVED** [review]
@GOODBOY008 Thanks for the update! LGTM

**Pull request merged**